pkg/commands/render.go

```diff
@@ -60,7 +60,7 @@ func NewRenderAPICommand(commonOpts *CommonOptions, opts *renderOptions) *cobra.
 			if commonOpts.UserPlatform == platform.Unknown {
 				return fmt.Errorf("must explicitely select a cluster platform")
 			}
-			apiManifests, err := api.GetManifests(commonOpts.UserPlatform)
+			apiManifests, err := api.GetManifests(commonOpts.UserPlatform, commonOpts.UpdaterType)
```

Comment:
it's suspicious that the API needs to know which updater is going to be used. Smells of convoluted design.

Reply:
This is temporary. Actually, both NFD and RTE should have the same API, but we didn't RTE didn't catch up with NFD latest changes. I also added a comment here about that: https://github.com/k8stopologyawareschedwg/deployer/blob/c8173b1f688e0fa2e04cac34a5c6c836ad8a0a4c/pkg/manifests/manifests.go#L151

Anyway, we can w8 w/ this PR till RTE will sync.

Reply:
yes, we should indeed wait and not mix APIs - especially considering they are unversioned.